eng/common/cross/android/arm/toolchain.cmake

```diff
@@ -0,0 +1,41 @@
+set(CROSS_NDK_TOOLCHAIN $ENV{ROOTFS_DIR}/../)
```

Comment:
For another PR, we need to use these and remove our Cross-build files.

Reply:
Ok.